rustic-popup.el

```diff
@@ -50,6 +50,9 @@ The first element of each list contains a command's binding."
 (defvar rustic-popup-buffer-name "rustic-popup-buffer"
   "Buffer name for rustic popup buffers.")
 
+(defvar rustic--popup-rust-src-name nil
```

Comment:
Why the extra "-" ?

Reply:
Does this make the variable local to rustic ? I've only seens this with `rustic-popup--`.

Reply:
I obviously haven't used this enough :D

Reply:
Yeah, mostly a convention to indicate that it's private. Do you want me to change it to `rustic-popup--` ? No strong opinions from my side.

Reply:
I think it makes more sense to keep it this way because it's also used in another file.